src/solutions/how-tos/amazon-msk/index.md

```diff
@@ -17,4 +17,10 @@ The IoT Ingest and Control MQTT Broker that lets clients publish messages and su
 
 The Secure Public Access Proxy lets authorized Kafka clients connect, publish messages and subscribe to topics in your Amazon MSK cluster via the internet.
 
-> [Amazon MSK Secure Public Access Proxy Guide](./secure-public-access/overview.md)
+> [Amazon MSK Secure Public Access Proxy Guide](./secure-public-access/production.md)
```

Comment:
Shouldn't this link to the overview page instead, same as where we link to from AWS Marketplace (shared) usage instructions across the different CloudFormation templates in the Zilla Plus for Amazon MSK product listing?

Reply:
This page is the new overview page for all of the deployment options and the user will get the context for different auth methods when clicking into the guide:

![image](https://github.com/user-attachments/assets/97063674-88b7-4f09-b6b0-dae5efd1bf94)

I will add the additional options to this section so the user has more visibility on the different options.